PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -784,6 +1289,64 @@ HTREEITEM ProjectPanel::addFolder(HTREEITEM hTreeItem, const TCHAR *folderName)
 	return addedItem;
 }
 
+HTREEITEM ProjectPanel::addDirectory(HTREEITEM hTreeItem, const TCHAR *folderName, bool root, const TCHAR *monitorPath, const std::vector<generic_string>* filters)

```

Comment:
So, `filters` is an optional parameter? Probably a good idea to mark it `_In_opt_`.